### ISSUE TITLE
Upload field thumbnail and filename links broken

### ISSUE THREAD

**Issue by 58bits** [payloadcms/payload#8103]
### Environment Info

```text
Payload Beta.100
```

### Describe the Bug

Upload field thumbnails no long appear, and links to upload file names are broken. 

We're now getting links like the following for upload fields...

https://beta.infonomic.iohttps//cdn.infonomic.io/photos/AGB_5882.jpg

http://localhost:3000https://cdn.forru.org/photos/forru-0001-0000199.jpg

Possibly related to https://github.com/payloadcms/payload/issues/8099 ?

Note that in our setup we are using a custom `getAdminThumbnail` function for our upload collections - as in:

```ts
const getAdminThumbnail = ({ doc }: any): string | null => {
  const mimeType = doc.mimeType as string
  if (mimeType === 'image/svg+xml') {
    return doc.url
  } else if (mimeType.includes('video')) {
    return `${process.env.S3_CDN_DOMAIN}/assets/thumbnail-video.jpg`
  } else if (mimeType.includes('audio')) {
    return `${process.env.S3_CDN_DOMAIN}/assets/thumbnail-audio.jpg`
  } else if (doc?.sizes?.thumbnail != null) {
    return doc.sizes.thumbnail.url
  } else {
    return null
  }
}
```

Which is then used in the collection configuration as...

```ts
upload: {
    // ignored when s3Storage is enabled
    staticDir: path.resolve(__dirname, '../../../../../uploads/photos'),
    adminThumbnail: getAdminThumbnail,
    mimeTypes: ['image/*'],
    ...
```
    
The above is working fine when viewing the collection in the admin dashboard.

We're also using `s3Storage` - with collections configured as....

```ts
  photos: {
          prefix: 'photos',
          // @ts-ignore
          generateFileURL: ({ collection, filename, prefix, size }) => {
            return `${process.env.S3_CDN_DOMAIN}/${prefix}/${filename}`
          }
        },
```

Below is a screenshot from our Photo block with the missing thumbnail and broken filename link (although this problem appears in all of our Upload field usage) - which produced the broken URLs above.

<img width="720" alt="image" src="https://github.com/user-attachments/assets/62309b31-d36f-422d-ad39-5fd5b8c0a7e9">

This bug appeared over the last two weeks and coincides with some code commits to the upload field.

Hope some of this helps, and apologies in advance for not including a reproducible repo.

### Reproduction Steps

As above.

**Comment by paulpopus** [comment]
@tylandavis did your PR fix this one as well?

**Comment by tylandavis** [comment]
@paulpopus no my PR was only for the bulk upload rows in the sidebar. I think this is related to the new upload UI that we built recently, there was probably a change somewhere with how we are handling thumbnail URLs.

**Comment by paulpopus** [comment]
Ill take it then

**Comment by tsemachh** [comment]
This also happens for us since this PR , it's a regression.

**Comment by paulpopus** [comment]
This will be fixed in https://github.com/payloadcms/payload/pull/8124

**Comment by 58bits** [comment]
> This will be fixed in #8124

Thanks @paulpopus
